### PR TITLE
Fix quoting issue in usage help text

### PR DESCRIPTION
**Pull request by alebcay**
#### What type of PR is this?

/kind bug

#### What this PR does / why we need it:

When using the backtick/grave (`) in the usage string for flags as a quote symbol, the quoted text is used as the placeholder and the instance in the usage string becomes unquoted.

Use single quote (') instead where this placeholder processing behavior is not intended.

Implementation of this behavior is from the urfave/cli package, see: https://github.com/cri-o/cri-o/blob/a2da00682712bf72a7a4ca5fd53da443c20a297e/vendor/github.com/urfave/cli/v2/flag.go#L228-L243

#### Which issue(s) this PR fixes:

Fixes https://github.com/cri-o/cri-o/issues/6230.

#### Special notes for your reviewer:

None

#### Does this PR introduce a user-facing change?

```release-note
Usage text for CLI flags --read-only, --bind-mount-prefix, --no-pivot, and --internal-wipe no longer display an erroneous placeholder value following the flag
```

### CODE REVIEW

**Comment by haircommander** [comment]
docs validation is failing. Your text editor added a newline to docs/crio.8.md. I can give tips on how to do so with vi
```
vi docs/crio.8.md
:set noeol
:set binary
:x
```
should do the trick

**Comment by alebcay** [comment]
Yeah, my VS Code added the extra newline. Fixed by `truncate`ing the file.

**Comment by haircommander** [comment]
/lgtm

thanks @alebcay

File: docs/crio.8.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
...
+**--bind-mount-prefix**="": A prefix to use for the source of the bind mounts. This option would be useful if you were running CRI-O in a container. And had '/' mounted on '/host' in your container. Then if you ran CRI-O with the '--bind-mount-prefix=/host' option, CRI-O would add /host to any bind mounts it is handed over CRI. If Kubernetes asked to have '/var/lib/foobar' bind mounted into the container, then CRI-O would bind mount '/host/var/lib/foobar'. Since CRI-O itself is running in a container with '/' or the host mounted on '/host', the container would end up with '/var/lib/foobar' from the host mounted in the container rather then '/var/lib/foobar' from the CRI-O container.
```
Comment (TomSweeneyRedHat) [inline docs/crio.8.md:152]: IDK, I think I prefer having it the old way for the md pages at least.  I think the back ticks(\`), make the quoted items stand out before than the single quote (\').  

For comparison:

And had '/' mounted on '/host' in your container.

vs:

And had `/` mounted on `/host` in your container.

I think it looks better when looking at a web-based man page anyway.  My $.02.

**Comment by TomSweeneyRedHat** [comment]
I'd vote not to change the md files to single quotes, but just my opinion.  If it's decided to move forward with this, the changes as intended look fine.

**Comment by alebcay** [comment]
I agree that would be preferable for the markdown. To be clear, the change only needs to be made in the actual help text, where `urfave/cli` handles strings containing backtick in the string differently.

The changes to the markdown are to satisfy the test for consistency between help text output and markdown.

**Comment by sohankunkerkar** [comment]
@alebcay could you rebase this PR on top of the main branch?

**Pull request merged**